### ISSUE TITLE
two 

### ISSUE THREAD

**Issue by liubinbinbin** [xxgege/StableNet#2]
1.the A is current features, the B is pre_features?
2.loss = criterion(output, target).view(1, -1).mm(weight1).view(1)，when  loss.backward()， the weight1 will be back propagation?

**Comment by xxgege** [comment]
1. No, actually A and B are two variables and they can be considered as two features in current features.
2. weight1 is only used for reweighting the loss calculated by each sample so that its update is not included in the backpropagation of the main network. The backpropagation of the main network only includes parameters of the main model.
